build.gradle.kts

```diff
@@ -271,7 +271,6 @@ dependencies {
     implementation(enforcedPlatform(libs.cel))
     implementation(libs.cel.core)
     implementation(libs.guava)
```

Comment:
We actually can't remove Guava. It's used in a bunch of places after looking more in-depth.